### ISSUE TITLE
Pass callback to learner of fit function?

### ISSUE THREAD

**Issue by lgvaz** [lgvaz/fastcook#1]
Explain the difference of passing a callback to the learner or to the fit function.

Callbacks passed to the learner are also applied to the validation set. Callbacks passed to the fit function are only applied to the training set.
